bmaptools/TransRead.py

```diff
@@ -55,7 +56,44 @@
 # A list of supported compression types
 SUPPORTED_COMPRESSION_TYPES = ('bz2', 'gz', 'xz', 'lzo', 'lz4', 'tar.gz',
                                'tar.bz2', 'tar.xz', 'tar.lzo', 'tar.lz4',
-                               'zip')
+                               'zip', 'tar')
+
+# Allow some file extensions to be recognised as aliases of others
+EXTENSION_ALIASES = {
+    '.tbz2': '.tar.bz2',
+    '.tbz':  '.tar.bz2',
+    '.tb2':  '.tar.bz2',
+    '.tgz':  '.tar.gz',
+    '.txz':  '.tar.xz',
+    '.tzo':  '.tar.lzo',
+    '.tlz4': '.tar.lz4',
+    '.gzip': '.gz',
+}
+
+# These need to be checked *before* those in EXTENSION_ALIASES
+NESTED_EXTENSION_ALIASES = {
+    '.tar.gzip': '.tar.gz',
+}
+
+# Info on how to decompress different archive formats
+DECOMPRESSER_EXTENSIONS = {
+    '.bz2': {'type': "bzip2", 'program': "bzip2",  'args': "-d -c"},
+    '.gz':  {'type': "gzip",  'program': "gzip",   'args': "-d -c"},
```

Comment:
Nitpick: would be nice to either use single or double quotes, not mix them in a seemingly random way.

I suggest to use double quotes everywhere.